.circleci/config.yml

```diff
@@ -57,9 +57,18 @@ jobs:
             wget https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.3-linux-x86_64.tar.gz
             tar xfz julia-*-linux-x86_64.tar.gz
             ln -s $(pwd)/julia-*/bin/julia /usr/bin/julia
-            conda update -y conda > /logs/conda.txt 2>&1
-            conda env update >> /logs/conda.txt 2>&1
+            # force install conda 4.7.11; circleCI has 4.6.14 and it finds it difficult to update
```

Comment:
I think the changes to this file are outdated

Reply:
Will checkout the current version